rust/azure_iot_operations_mqtt/Cargo.toml

```diff
@@ -14,7 +14,7 @@ publish = true
 [dependencies]
 anyhow = "1.0.86"                                                                   # TODO: reconsider inclusion once TLS library is finalized
 async-trait = "0.1.81"
-azure_mqtt = { version = "0.1.0", path = "../../../mqtt-client" } # TODO: Because of authentication issues, including the dependency this way for now.
+azure_mqtt = { git = "https://github.com/Azure/mqtt-client.git", rev = "e88fc19c56e65878347eee7bad508a8e8607085b" }
```

Comment:
I changed this to the hash just so it's clearer what version checked in code is against since the mqtt client is evolving rapidly